src/main/java/uk/gov/dvsa/model/cvs/certificateData/MsvaFailCertificateData.java

```diff
@@ -0,0 +1,128 @@
+package uk.gov.dvsa.model.cvs.certificateData;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MsvaFailCertificateData {
+    @JsonProperty("serialNumber")
+    private String serialNo;
```

Comment:
Payload from cert-gen is 

vehicleZNumber

Reply:
Appreciate the @JsonProperty is mapping from the same, but for consistency with the rest of this class - I wonder if it's worth just changing the variable to serialNumber too?